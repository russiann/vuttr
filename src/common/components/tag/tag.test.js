import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Tag from './tag.component';

afterEach(cleanup);

it('snapshot', () => {
  const {asFragment} = render(<Tag>#node</Tag>);
  expect(asFragment()).toMatchSnapshot();
});
