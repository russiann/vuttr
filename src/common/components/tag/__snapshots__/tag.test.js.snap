// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <span
    class="css-1f19l54-Tag ex7fybt0"
  >
    #node
  </span>
</DocumentFragment>
`;

exports[`snapshot 1`] = `
<DocumentFragment>
  <span
    class="css-1f19l54-Tag ex7fybt0"
  >
    #node
  </span>
</DocumentFragment>
`;
